src/js/edu-benefits/5490/config/form.js

```diff
@@ -93,53 +93,13 @@ const formConfig = {
     applicantInformation: {
       title: 'Applicant Information',
       pages: {
-        applicantInformation: applicantInformation(fullSchema5490, {
+        applicantInformation: applicantInformationPage(fullSchema5490, {
```

Comment:
Yeah, I'm not sure how much it really matters, but I figured I'd touch up as I went along.